cmd/bnsd/app/datamigration.go

```diff
@@ -263,21 +263,34 @@ func rewriteAccountBlockchainIDs(ctx context.Context, db weave.KVStore) error {
 // See https://github.com/ChainAgnostic/CAIPs/tree/master/CAIPs
 func migrateAccountTargetBlockchainID(targets []account.BlockchainAddress) ([]account.BlockchainAddress, bool) {
 	var updated bool
+	// this is a set of old chain ID to new chain ID
+	var oldToNewChainID = map[string]string{
+		"ethereum-eip155-1":           "eip155:1",
+		"iov-mainnet":                 "cosmos:iov-mainnet",
```

Comment:
@davepuchyr asking out of curiosity: can you explain why iov-mainnet chain id starts with `cosmos:` prefix?

Reply:
I don't know how that came to be other than it was agreed upon by the https://github.com/ChainAgnostic/CAIPs group, which was largely driven by Antoine and Simon.